osquery/tables/system/windows/deviceguard_status.cpp

```diff
@@ -42,18 +42,18 @@ QueryData genHVCIStatus(QueryContext& context) {
     long vbs_status;
     data.GetLong("VirtualizationBasedSecurityStatus", vbs_status);
     r["vbs_status"] =
-        vbs_methods.size() < vbs_status ? vbs_methods[vbs_status] : "UNKNOWN";
+        vbs_methods.size() > vbs_status ? vbs_methods[vbs_status] : "UNKNOWN";
```

Comment:
Is this logic change intentional?

Reply:
Yes it is. Unfortunately the logic error appears to have been present since the initial commit of this module. The size of the vector should always be greater than the status value when the value is known. I have verified that the current release has broken behavior on these statuses as well.